src/commands/games/2048.js

```diff
@@ -0,0 +1,155 @@
+      if ((Date.now() - lastPlay) / 1000 / 60 >= 2) {
+        collector.stop()
+        msg.reactions.removeAll().catch(function () {})
+        msg.edit(`:skull:${this.drawPoints(points)}\n\n${this.drawBoard(board)}`)
+        clearInterval(interval)
+      }
+    }, 1000 * 60)
+    collector.on('collect', (r) => {
+      const u = r.users.cache.last()
+      if (u.id !== author.id) return
+      const old = Array.from(board.reduce((acc, val) => acc.concat(val), []))
+      switch (r.emoji.name) {
+        case arrows[0]:
+          this.rotateBoard(board, true)
+          this.rotateBoard(board, true)
+          points += this.slideLeft(board)
+          this.rotateBoard(board, false)
+          this.rotateBoard(board, false)
+          break
+        case arrows[1]:
+          this.rotateBoard(board, false)
+          points += this.slideLeft(board)
+          this.rotateBoard(board, true)
+          break
+        case arrows[2]:
+          this.rotateBoard(board, true)
+          points += this.slideLeft(board)
+          this.rotateBoard(board, false)
+          break
+        case arrows[3]:
+          points += this.slideLeft(board)
+      }
+      if (board.reduce((acc, val) => acc.concat(val), []).some((tile, i) => tile !== old[i])) {
+        const empty = this.getEmptyTile(board)
+        if (empty !== false) {
+          board[Math.floor(empty / size)][Math.floor(empty % size)] = 2
+        }
+        lastPlay = Date.now()
+        msg.edit(`${this.drawPoints(points)}\n\n${this.drawBoard(board)}`)
+      }
+      r.users.remove(author).catch(function () {})
+    })
+  }
+  slideLeft (board) {
+    let points = 0
+    for (let i = 0; i < board.length; i++) {
+      let s = 0
+      for (let j = 1; j < board[i].length; j++) {
+        for (let k = j; k > s; k--) {
+          if (board[i][k - 1] === 0) {
+            board[i][k - 1] = board[i][k]
+            board[i][k] = 0
+          } else if (board[i][k - 1] === board[i][k]) {
+            points += board[i][k] * 2
+            board[i][k - 1] += board[i][k]
+            board[i][k] = 0
+            s = k
+            break
+          } else {
+            break
+          }
+        }
+      }
+    }
+    return points
+  }
+  rotateBoard (board, option) {
+    const rotatedBoardArray = this.createBoard(board.length)
+    for (let i = 0; i < board.length; i++) {
+      for (let j = 0; j < board[i].length; j++) {
+        if (option) {
+          rotatedBoardArray[i][j] = board[j][board.length - i - 1]
+        } else {
+          rotatedBoardArray[i][j] = board[board.length - j - 1][i]
+        }
+      }
+    }
+    for (let i = 0; i < board.length; i++) {
+      board[i] = rotatedBoardArray[i]
+    }
+  }
+  drawBoard (board) {
+    let text = ''
+    for (const row of board) {
+      for (const tile of row) {
+        const stringTile = tile.toString()
+        text += this.getEmoji(stringTile.length === 1 ? stringTile + '_' : stringTile)
```

Comment:
Just to match how we've saved the emoji:
```suggestion
        text += this.getEmoji(`${stringTile}_`)
```